docs/naps/6-contributable-menus.md

```diff
@@ -0,0 +1,517 @@
+:orphan:
+
+(nap-template)=
```

Comment:
I think this needs to be changed?

Reply:
@brisvag yep fixed this and confirmed it now shows up in TOC.